Federicor/progs for pneumtaics

federrg opened the pull request:
Add programs in teh Application_Specific fodler for:
- Pneumatic box
- Safety Shutter
Update templates for:
- Pneumatics Axes
- Shutter Operator Panel

federrg committed:
Add new programs and templates for penaumtics

Add a template for safety shutter, wher ethe PSS permit signals are created.
Add a program for the Pneumatic_Box, where all the signals for pressure sensors
and pressure calcualtions are implemented.
Correct the Pneumatics_Template TaskInfo was missing a _, delete the pressurelimtis,
and add the pressure group selection.
Call the new programs from PROG() MAIN.

federrg committed:
Rename Cabinet_Configuration to Cabinet_Monitoring

Remove the pneumatics variables from Cabinet_Configuration and move them to
Pneumatic_Box program.Because of that the program now handles onle the
cabinet monitoring. Rename the prgram to reflect that change
deleted:    solution/tc_project_app/POUs/Application_Specific/Applications/Cabinet_Configuration.TcPOU
new file:   solution/tc_project_app/POUs/Application_Specific/Applications/Cabinet_Monitoring.TcPOU

federrg committed:
Remove the negation(NOT) from the PSS permit status signal

federrg committed:
Add as constant variable the tTIME_PRESSURE_OUT_OF_RANGE variable

federrg committed:
Build solution with the new pneumatics progs and variables

federrg committed:
Fix indentation and typo in comment

federrg committed:
Add new lines for configurable variables fromg GVL_APP

federrg committed:
Program for Safety Shutters

Program includes all the needed Inputs and Outputs for the PSS signals
related to the Safety shutter.
Also contains all the needed configurable variables.
modified:   solution/tc_project_app/POUs/Application_Specific/Axes/Safety_Shutter.TcPOU

federrg committed:
Rename the Shutter operator panel program and update the PSS permit variable link

Add the comment to call the Shutter_Operator_Panel program from MAIN/PROG()
Build solution.
renamed:    solution/tc_project_app/POUs/Application_Specific/Applications/Shutter_Operator_panel_Template.TcPOU -> solution/tc_project_app/POUs/Application_Specific/Applications/Shutter_Operator_Panel.TcPOU
modified:   solution/tc_project_app/POUs/MAIN.TcPOU
modified:   solution/tc_project_app/tc_project_app.plcproj

federrg committed:
Build solution including the fix of typo in comment

federrg committed:
Add initial values to low and high pressure limits values, fix indentation

modified:   solution/tc_project_app/GVLs/GVL_APP.TcGVL

federrg committed:
Update pointer to tc_mca_Std_lib to MBP-310 commit

KJurisicESS commented on solution/tc_project_app/POUs/Application_Specific/Applications/Pneumatic_Box.TcPOU:
```diff
@@ -0,0 +1,104 @@
+    <Declaration><![CDATA[PROGRAM Pneumatic_Box
+VAR
+    //Define all necessary variables for your application
+    bPowerPressureSensors AT %Q*: BOOL:= TRUE; //Variable to power the air pressure sensors in the pneuatmic box
```
We don't want to add the variable to power the LED on the pneumatic box?

federrg committed:
Extend the multiline comment to include the PSSPermit code

federrg committed:
Add output variable to power the green LED and the inside light of box, build solution

modified:   solution/_Config/PLC/tc_project_app.xti
modified:   solution/tc_project_app/POUs/Application_Specific/Applications/Pneumatic_Box.TcPOU

federrg committed:
Update VISU text elements and delete unused elements

federrg committed:
Update pointer to tc_mca_std_lib to commit cd60a1f9

federrg committed:
Update pointer of std_lib to the merge to master of MBP-310
